rotkehlchen/tests/unit/test_evm_names.py

```diff
@@ -132,22 +134,34 @@ def test_download_ens_avatar(ethereum_inquirer):
             name='nebolax.eth',
             now=ts_now(),
         )
+        dbens.add_ens_mapping(
+            write_cursor=write_cursor,
+            address=make_evm_address(),
+            name='tewshi.eth',
+            now=ts_now(),
+        )
     with tempfile.TemporaryDirectory() as tempdir_str:
         tempdir = Path(tempdir_str)
         try_download_ens_avatar(
             eth_inquirer=ethereum_inquirer,
             avatars_dir=tempdir,
             ens_name='random.ens.name.eth',  # a random ens name, and thus there is no avatar
         )
-        assert dbens.get_last_avatar_update('random.ens.name.eth') == ts_now(), 'Last update timestamp should have been set'  # noqa: E501
+        assert dbens.get_last_avatar_update('random.ens.name.eth') <= ts_now(), 'Last update timestamp should have been set'  # noqa: E501
         assert list(tempdir.iterdir()) == []
         try_download_ens_avatar(
             eth_inquirer=ethereum_inquirer,
             avatars_dir=tempdir,
-            ens_name='nebolax.eth',  # an avatar should be downloaded
+            ens_name='nebolax.eth',  # an avatar should be downloaded. Resolver v2.
+        )
+        assert dbens.get_last_avatar_update('nebolax.eth') <= ts_now(), 'Last update timestamp should have been set'  # noqa: E501
+        try_download_ens_avatar(
+            eth_inquirer=ethereum_inquirer,
+            avatars_dir=tempdir,
+            ens_name='tewshi.eth',  # an avatar should be downloaded. Resolver v3
         )
-        assert dbens.get_last_avatar_update('nebolax.eth') == ts_now(), 'Last update timestamp should have been set'  # noqa: E501
-        assert list(tempdir.iterdir()) == [tempdir / 'nebolax.eth.png']
+        assert dbens.get_last_avatar_update('tewshi.eth') <= ts_now(), 'Last update timestamp should have been set'  # noqa: E501
+        assert list(tempdir.iterdir()) == [tempdir / 'tewshi.eth.png', tempdir / 'nebolax.eth.png']
```

Comment:
Did you run it locally? Wonder why the order is different if so

Reply:
Becuse it's a different system and order of files is not guaranteed by iterdir